### DISCUSSION TITLE
Time availablity windows

### DISCUSSION THREAD

**Discussion by kbzowski** [lukevella/rallly#374] [Ideas]
I would like to propose a new way of defining meetings. It is often difficult to define slots (as it works now) for a meeting of very busy teams. It appears to me that it would be a good idea if everyone could declare their availability time (in an arbitrary way) and then the owner could choose the overlapping time frames.
The way I see it: the owner states that he is available between 8-12 and 15-18 on selected days and needs to do a 1h meeting (of course on any day he could give any number of such ranges). All invitees define their availability windows in a similar way - as any number of time spans in which they are available. Ultimately, the owner chooses the time period that suits the largest number of people (similar to current situation), could even opt for a shorter meeting. With this approach, there is no need to create rigid meeting slots and it would be possible, for example, to start a meeting 8:10. To do something similar at the moment I have to create multiple overlapping slots every 10-15 minutes.

**Comment by lukevella** [comment]
Hi @kbzowski , I see this as a different solution to a similar problem, not as a replacement for what is currently offered. I wouldn't exclude adding it as an addition to meeting polls within Rallly but it's not currently part of the roadmap. In the meantime, I'd recommend checking out [crab.fit](https://crab.fit) which takes the approach you are describing.

**Comment by kbzowski** [comment]
@lukevella In no way am I suggesting that existing functions should be replaced. I was more thinking of it as a proposal for additional functionality. CrabFit doesn't compare to Rallly's look. Not that league. You have done a great job on this app.  I see that it also has restrictions to one range.
